community/kernel/src/main/java/org/neo4j/kernel/impl/logging/StoreLogService.java

```diff
@@ -83,6 +85,12 @@ public Builder withRotationListener( Consumer<LogProvider> rotationListener )
             return this;
         }
 
+        public Builder withLevel( FormattedLog.Level level )
```

Comment:
Perhaps consider `StoreLogService.InternalLogLevel` rather than exposing `FormattedLog` here, as it is just the currently chosen implementation of the log.